andela/pippin-ah-backend
# #163114860 Create Report Model

#### What does this PR do?
Create a Report Model and Migration

#### Description of Task to be completed?
- Create `report` migration flie in `migration` directory and add the migrations.
- Create `report` model file in `model` directory and add the model.

#### How should this be manually tested?
- run `npm run db:migrate`
- check `database` to see `report` table amongst table created.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
#163114860

#### Screenshots (if appropriate)

#### Questions:
